genie-proto/src/main/proto/genie.proto

```diff
@@ -335,8 +336,10 @@ message ServerControlMessage {
 message ServerFileRequestMessage {
     string stream_id = 1;
     string relative_path = 2;
-    int32 start_offset = 3;
-    int32 end_offset = 4;
+    int32 deprecated_start_offset = 3; // Deprecated
+    int32 deprecated_end_offset = 4;   // Deprecated
+    int64 start_offset = 5;
+    int64 end_offset = 6;
```

Comment:
This is good however protobuf has support for [deprecating fields](https://developers.google.com/protocol-buffers/docs/proto3#options) as well:
```deprecated (field option): If set to true, indicates that the field is deprecated and should not be used by new code. In most languages this has no actual effect. In Java, this becomes a @Deprecated annotation. In the future, other language-specific code generators may generate deprecation annotations on the field's accessors, which will in turn cause a warning to be emitted when compiling code which attempts to use the field. If the field is not used by anyone and you want to prevent new users from using it, consider replacing the field declaration with a reserved statement.```

Might add some extra value to the generated code.

Reply:
Good call, I'll add it.